airflow/example_dags/example_trigger_target_dag.py

```diff
@@ -62,12 +59,13 @@ def run_this_func(ds, **kwargs):
     task_id='run_this',
     provide_context=True,
     python_callable=run_this_func,
-    dag=dag)
-
+    dag=dag,
+)
 
 # You can also access the DagRun object in templates
 bash_task = BashOperator(
     task_id="bash_task",
     bash_command='echo "Here is the message: '
-                 '{{ dag_run.conf["message"] if dag_run else "" }}" ',
-    dag=dag)
+    '{{ dag_run.conf["message"] if dag_run else "" }}" ',
```

Comment:
I prefer the original indentation.

Reply:
My auto formatter (Black) done it that way. Will correct.